Chunks/ChunkManagement/ChunkWorkers/NonGenerativeChunkWorker.cs

```diff
@@ -1,11 +1,12 @@
 ﻿using System.Collections.Concurrent;
```

Comment:
In this file its mosly just changing the order of methods + making it so that chunks are generated before exiting constructor so that when we try to spawn bots chunks are already there